stimuli-ci-generation/Dockerfile_base

```diff
@@ -0,0 +1,21 @@
+FROM si_rocker_1
+
+RUN apt-get update 
+
+
+RUN apt-get install -y --no-install-recommends python3.7 
+
+RUN apt-get install -y --no-install-recommends python3-pip 
+
+RUN apt-get install -y --no-install-recommends python3-setuptools 
+
+RUN apt-get install -y --no-install-recommends python3-dev
+
+WORKDIR /app
+
+COPY requirements.txt /app/requirements.txt
+
+RUN pip3 install -r requirements.txt
+
+RUN pip3 install google-cloud-storage
```

Comment:
you can put google-cloud-storage inside of `requirements.txt` here

Reply:
It's already in the requirement file. This would be one of the few redundancies that need to be removed.